app/assets/javascripts/datepickers.js

```diff
@@ -47,6 +47,32 @@ function load_datepicker() {
       $('.date_end_no_min').datepicker( "option" , "minDate" , start_date);
     }
   });
-};
 
+  $('.date_checkout').datetimepicker({
+    dateFormat: "mm/dd/yy",
+    timeFormat: "h:mm TT",
+    altFieldTimeOnly: false,
+    altFormat: "yy-mm-dd",
+    altTimeFormat: "HH:mm",
+    altField: '#date_checkout_alt',
+    minDate: 0,
```

Comment:
Yes the minDate is what's preventing from choosing a past date. I removed it and the check will based on the JS function I'll write later